src/bundle/Controller/ContentViewController.php

```diff
@@ -449,4 +469,41 @@ private function supplyIsLocationBookmarked(ContentView $view): void
 
         $view->addParameters(['location_is_bookmarked' => $locationIsBookmarked]);
     }
+
+    /**
+     * @param \eZ\Publish\Core\MVC\Symfony\View\ContentView $view
+     *
+     * @throws \eZ\Publish\API\Repository\Exceptions\BadStateException
+     * @throws \eZ\Publish\API\Repository\Exceptions\InvalidArgumentException
+     * @throws \eZ\Publish\API\Repository\Exceptions\NotFoundException
+     */
+    private function supplyContentCreatePermissionsForMFU(ContentView $view): void
```

Comment:
You are hiding quite complicated logic in controller. Better place for your code is in `\EzSystems\EzPlatformAdminUi\UI\Config\Provider\Module\SubItemsList`